src/2d/Node2D.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include "en/Node.hpp"
 #include "sx/math.h"
+#include "sx/macros.h"
```

Comment:
Why did you include sx/macros.h?

Reply:
I was trying to solve the last error, but I've forgotten to delete this include!

Reply:
fixed